internal/validation/errors_test.go

```diff
@@ -24,7 +55,7 @@ func TestNewValidationErrorSingleField(t *testing.T) {
 	}
 
 	doc, field := references(t, st, &st.OtherField, "")
-	err := newError(doc, field, assert.AnError)
+	err := newTraceError(doc, field, assert.AnError)
```

Comment:
lets call it traceError or treeError. I advocate trace

Reply:
Those 2 do a different thing. TreeError formats a tree of errors nicely, while trace-error traverses the struct to find the field